platform/core/src/services/MeasurementService/MeasurementService.js

```diff
@@ -612,7 +619,7 @@ class MeasurementService {
    *
    * @param {string} eventName The event name
    * @param {MeasurementSource} source The measurement source
-   * @param {string} measurement The measurement id
+   * @param {string} measurement The measurement
```

Comment:
It's not a String, it's an Object, isn't it?

Reply:
Good catch, updated, thanks